reposerver/repository/repository.go

```diff
@@ -211,7 +211,7 @@ func appPath(root, path string) (string, error) {
 }
 
 // GenerateManifests generates manifests from a path
-func GenerateManifests(root, path string, q *apiclient.ManifestRequest) (*apiclient.ManifestResponse, error) {
+func GenerateManifests(root, path string, q *apiclient.ManifestRequest, commitSHA string) (*apiclient.ManifestResponse, error) {
```

Comment:
Not sure why you need to add revision here. Why not examine `.ManifestRequest.Revision`?

Reply:
The problem here is that I believe `.ManifestRequest.Revision` may only reference things like `HEAD`, or other human-friendly things.

Instead, I did it like this so that when the commit hash of whatever human-friendly thing you are pointed at (`head`, `test`, `staging`, etc) changes, you the manifest changes, then triggering a sync.

I'm pre-coffee ATM, so hope that made sense :)

Reply:
Ok. I got it. I think that manifest request should take the resolved revision. Two revisions would cause confusion.

Reply:
You're correct. Can I request two changes:

* Rename `commitSHA` to `resolvedRevision`.
* Move `resolvedRevision` to be the third parameter.